NEWS.md

```diff
@@ -26,6 +27,113 @@ The format is based on [Keep a Changelog]
+
+### Added
+
+- Expose output configuration test to clients. For example, this enables
+  `wlr-randr --dryrun`
+- Support mapping of touch devices by name to output (display).
+  Written-by: @jp7677
+- Add window-edge resistance for interactive moves/resizes and support negative
+  strengths to indicate attractive snapping. Written-by: @ahesford
+
+      <resistance>
+        <screenEdgeStrength>-20</screenEdgeStrength>
+        <windowEdgeStrength>-20</windowEdgeStrength>
+      </resistance>
+
+- Set keyboard layout on reconfigure. Issue #1407
+- Reset keyboard-layout group (index) for each window on reconfigure if
+  the keymap has changed.
+- Support merging multiple config files with the --merge-config command
+  line option. Issue #1406
+- Add config option to map touch events to output names.
+  Written-by: @jp7677
+
+      <touch mapToOutput=""/>
+
+- Add tablet support including:
+    - Mapping of tablet to output (display)
+    - Emulation of cursor movement and button press/release
+    - Configuration of area and rotation
+  Written-by: @jp7677 @Consolatis
+
+      <tablet mapToOutput="HDMI-A-1" rotate="90">
+        <area top="0.0" left="0.0" width="0.0" height="0.0" />
+        <map button="Tip" to="Left" />
+        <map button="Stylus" to="Right" />
+        <map button="Stylus2" to="Middle" />
+      </tablet>
+
+- Add tearing support. #1390. Written-by: @Ph42oN @ahesford
+- Add configuration support for mouse buttons `Side`, `Extra`, `Forward`,
+  `Back` and `Task`. Written-by: @jp7677
+- config: allow `<libinput><device>` without category attribute to define a
+  `default` profile because it is more user-friendly and intuitive.
+- Add a configuration option enable adaptive sync only when an application is
+  in fullscreen mode. Written-by: @Ph42oN
+- Add `touchpad` libinput device type to increase configuration flexibility,
+  for example allowing `naturalScroll` on touchpads, but not on regular pointer
+  devices such as mice. Written-by: @jmbaur
+- Add actions:
+    - `AutoPlace` (by @ahesford)
+    - `MoveToOutput`, `FitToOutput` (by @jp7677)
+    - `Shade`, `Unshade`, `ToggleShade` (by @ahesford @Consolatis)
+- Add config option `<placement><policy>` with supported values `center`,
+  `under-cursor` and `automatic`. The latter minimizes overlap with other
+  windows already on screen and is similar to Openbox's smart window placement.
+  Written-by: @ahesford #1312
+
+      <placement>
+        <policy>center|automatic|cursor</policy>
+      </placement>
+
+### Fixed
+
+- Fix output configuration bug causing compositor crash when refresh rate is
+  zero. Issue #1458
+- Fix disappearing cursor bug on view destruction. Issue #1393
+- Use used specified config-file (using -c command line option) on
+  reconfigure.
+- view: assign output on surface creation instead of mapping and notify
+  the client of the preferred output scale when doing so. This fixes an
+  issue with foot: https://codeberg.org/dnkl/foot/issues/1579
```

Comment:
```suggestion
- Assign outputs to new views on surface creation instead of mapping, and notify
  the client of the preferred output scale when doing so. This fixes an
  issue with foot: https://codeberg.org/dnkl/foot/issues/1579
```

Reply:
Done.